charts/orchestrator/templates/sonata-network-policy.yaml

```diff
@@ -0,0 +1,34 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: RHDH-to-sonata-runtimes
+spec:
+  podSelector:
+    matchExpressions:
+      - { key: sonataflow.org/workflow-app, operator: Exists }
+  ingress:
+    - from:
+      - namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: rhdh-operator 
+        podSelector:
+          matchLabels:
+            rhdh.redhat.com/app: backstage-backstage
+---
+
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: RHDH-to-sonata-services
```

Comment:
wdyt of replacing with `rhdh-to-sonataflow-workflows`?

Reply:
I think this NP is for the DI and JS, not the workflows